list-map/src/main/scala/examples/ListMapNoResume.scala

```diff
@@ -0,0 +1,32 @@
+package examples
+
+import continuations.Continuation
+import continuations.SafeContinuation
+import continuations.Suspend
+
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+import scala.util.Random
+
+given ExecutorService = Executors.newWorkStealingPool()
+
+@main def ListMapNoResume =
+  def zeroArgumentsZeroContinuations()(using Suspend): Int = 1
+
+  def twoArgumentsOneContinuationsCFBefore(
+      x: Int,
+      y: Int): Suspend ?=> Int | Continuation.State.Suspended.type =
+    println("twoArgumentsOneContinuationsCFBefore")
+    val z = 1
+    summon[Suspend].suspendContinuation[Int] { c =>
```

Comment:
this works now? nice 👏 
maybe we could rename the file?

Reply:
Yeah, I'll rename the example, good call.